src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+}
```

Comment:
Would be better to reset margin on `body` directly

Reply:
replaced